mne/report.py

```diff
@@ -1615,10 +1647,15 @@ def __exit__(self, type, value, traceback):
 
     @staticmethod
     def _gen_caption(prefix, fname, data_path, suffix=''):
-        if data_path is None:
-            caption = f'{prefix}: {fname} {suffix}'
-        else:
-            caption = f'{prefix}: {fname[len(data_path)+1:]} {suffix}'
+        if data_path is not None:
+            # Ensure we drop leading path separators from the
+            # filename before using it as a caption. We don't know whether
+            # `data_path` has a trailing separator or not; so create a Path
+            # just to be safe (trailing separators will be removed during the
+            # roundtrip).
+            fname = fname[len(str(Path(data_path))) + 1:]
```

Comment:
`os.path.relpath` instead?

Reply:
No it would have a different behavior than what I'm trying to achieve here… I know it's not pretty, but it does to the job.

Reply:
When would it be different?

Reply:
Never mind, `op.relpath` makes total sense here. Thanks!